less/wizard.less

```diff
@@ -10,178 +10,185 @@
 		background-color: @tableBackgroundAccent;
 		position: relative;
 		min-height: 48px;
-		overflow: hidden;
 
-		> .steps {
-			list-style: none outside none;
-			padding: 0;
-			margin: 0;
-			width: 4000px;
+		.steps {
+			border-radius: @baseBorderRadius @baseBorderRadius 0 0;
+			overflow: hidden;
 
-			&.previous-disabled {
-				li {
-					&.complete {
-						cursor: default;
+			ul {
+				list-style: none outside none;
+				padding: 0;
+				margin: 0;
+				width: 999999px;
```

Comment:
that was `width: 4000px;` before. I bumped this number to `999999` to give it a ridiculous amount of space for the li elements to flow horizontally